CHANGELOG.md

```diff
@@ -1,4 +1,10 @@
-1.35.0
+# 3.1.0
```

Comment:
Just for clarity, we plan for this to reflect the sdk version that the change / addition shipped in?

We may want to continue to use `3.1.0-wip` here then, until the sdk does ship, and we can retroactively remove the `-wip` from the latest entry and add a space for a new `3.(N+1).0-wip` entry. I'm not sure how this would all be handled through when the linter moves into the sdk - whether we still want a separate changelog or not.

Reply:
Works for me.  As for when the linter is in the SDK, we might still want this changelog but I'm totally open to deleting it.

Reply:
> Just for clarity, we plan for this to reflect the sdk version that the change / addition shipped in?

Sorry, yes. This is right.